components/card/examples/01-card-examples.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+} from '../src/index';
+
+export default () => (
+  <div>
+    <Card hasShadow={false}>
+      <CardHeader>
+        Header
+      </CardHeader>
+      <CardBody>
+        This card has now shadow below.
+      </CardBody>
+      <CardFooter>
+        Footer
+      </CardFooter>
+    </Card>
+
+    <br/>
+
+    <Card hasBorder={true}>
```

Comment:
This should probably be inverted so you can say <Card hasBorder />. This would mean the default should be it does not have a border or we should change the `hasBorder` to `noBorder` or something similar.